src/index.css

```diff
@@ -24,8 +24,8 @@ a:hover {
 
 body {
   margin: 0;
-  display: flex;
-  place-items: center;
+  /* display: flex;
+  place-items: center; */
```

Comment:
I commented these lines out because in the lesson they positioned their list in the left upper corner; so, I thought that we had to position our to-do list the same. I can uncomment these lines.